### PR TITLE
Fix more warnings

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs [hunk 1]
```diff
@@ -10,40 +10,44 @@
...
 {
     /// <summary>
-    /// Class UdpServerEntryPoint
+    /// Class UdpServerEntryPoint.
     /// </summary>
     public class UdpServerEntryPoint : IServerEntryPoint
     {
         /// <summary>
-        /// Gets or sets the UDP server.
+        /// The port of the UP server.
```
Comment (JustAMan) [inline Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs]:
```suggestion
        /// The port of the UDP server.
```

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 2]
```diff
@@ -367,6 +369,8 @@ public async Task<User> AuthenticateUser(string username, string password, strin
             return success ? user : null;
         }
 
+#nullable enable
```
Comment (JustAMan) [inline Emby.Server.Implementations/Library/UserManager.cs]: what this means?

Comment (Bond-009) [reply Emby.Server.Implementations/Library/UserManager.cs]: This does the same as #1942 but only for a specific region. In the end we would want all code to be covered by this. I'm just not sure how we should undertake that. We can either do it like #1942 whole projects at a time (by excluding all files and adding them in 1by one). Or the other way around, adding regions until the whole solution is covered.

File: Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs (hunk 1 above)
Comment (JustAMan) [reply Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs]: You seem to lose this comment fix :)

**Comment by Bond-009** [comment]
@JustAMan guess I forgot to pull.

File: Emby.Server.Implementations/Data/BaseSqliteRepository.cs [hunk 3]
```diff
@@ -11,6 +13,10 @@ public abstract class BaseSqliteRepository : IDisposable
     {
         private bool _disposed = false;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BaseSqliteRepository"/> class.
+        /// </summary>
+        /// <param name="logger">The ogger.</param>
```
Comment (dkanada) [inline Emby.Server.Implementations/Data/BaseSqliteRepository.cs]:
```suggestion
        /// <param name="logger">The logger.</param>
```

File: Emby.Server.Implementations/Configuration/ServerConfigurationManager.cs [hunk 4]
```diff
@@ -119,24 +127,32 @@ private void ValidateSslCertificate(BaseApplicationConfiguration newConfig)
...
+                            "{0} does not exist.",
+                            newPath));
                 }
 
                 EnsureWriteAccess(newPath);
             }
         }
 
+        /// <summary>
+        /// Sets all config values to the optimal value.
```
Comment (dkanada) [inline Emby.Server.Implementations/Configuration/ServerConfigurationManager.cs]:
```suggestion
        /// Sets all configuration values to their optimal values.
```

File: Emby.Server.Implementations/AppBase/BaseConfigurationManager.cs [hunk 5]
```diff
@@ -339,6 +350,11 @@ public void SaveConfiguration(string key, object configuration)
             OnNamedConfigurationUpdated(key, configuration);
         }
 
+        /// <summary>
+        /// Event handler for when a named configuration got updates.
```
Comment (dkanada) [inline Emby.Server.Implementations/AppBase/BaseConfigurationManager.cs]:
```suggestion
        /// Event handler for when a named configuration has been updated.
```

File: Emby.Server.Implementations/EntryPoints/RefreshUsersMetadata.cs [hunk 6]
```diff
@@ -12,42 +11,51 @@
...
-    /// Class RefreshUsersMetadata
+    /// Class RefreshUsersMetadata.
     /// </summary>
     public class RefreshUsersMetadata : IScheduledTask, IConfigurableScheduledTask
     {
         private readonly ILogger _logger;
+
         /// <summary>
-        /// The _user manager
+        /// The _user manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/RefreshUsersMetadata.cs]:
```suggestion
        /// The user manager.
```

File: Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs [hunk 7]
```diff
@@ -16,33 +16,46 @@
...
 {
     /// <summary>
-    /// Class WebSocketEvents
+    /// Class WebSocketEvents.
     /// </summary>
     public class ServerEventNotifier : IServerEntryPoint
     {
         /// <summary>
-        /// The _user manager
+        /// The _user manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs]:
```suggestion
        /// The user manager.
```

File: Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs [hunk 8]
```diff
@@ -16,33 +16,46 @@
...
         /// </summary>
         private readonly IUserManager _userManager;
 
         /// <summary>
-        /// The _installation manager
+        /// The _installation manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs]:
```suggestion
        /// The installation manager.
```

File: Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs [hunk 9]
```diff
@@ -16,33 +16,46 @@
...
         /// </summary>
         private readonly IInstallationManager _installationManager;
 
         /// <summary>
-        /// The _kernel
+        /// The _kernel.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs]:
```suggestion
        /// The kernel.
```

File: Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs [hunk 10]
```diff
@@ -16,33 +16,46 @@
...
         /// </summary>
         private readonly IServerApplicationHost _appHost;
 
         /// <summary>
-        /// The _task manager
+        /// The _task manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/ServerEventNotifier.cs]:
```suggestion
        /// The task manager.
```

File: Emby.Server.Implementations/EntryPoints/StartupWizard.cs [hunk 11]
```diff
@@ -8,31 +8,36 @@
...
 {
     /// <summary>
-    /// Class StartupWizard
+    /// Class StartupWizard.
     /// </summary>
     public class StartupWizard : IServerEntryPoint
     {
         /// <summary>
-        /// The _app host
+        /// The _app host.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/StartupWizard.cs]:
```suggestion
        /// The app host.
```

File: Emby.Server.Implementations/EntryPoints/StartupWizard.cs [hunk 12]
```diff
@@ -8,31 +8,36 @@
...
         /// </summary>
         private readonly IServerApplicationHost _appHost;
+
         /// <summary>
-        /// The _user manager
+        /// The _user manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/StartupWizard.cs]:
```suggestion
        /// The user manager.
```

File: Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs [hunk 13]
```diff
@@ -10,40 +10,44 @@
...
-        /// Gets or sets the UDP server.
+        /// The port of the UDP server.
         /// </summary>
-        /// <value>The UDP server.</value>
-        private UdpServer UdpServer { get; set; }
+        public const int PortNumber = 7359;
 
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/EntryPoints/UdpServerEntryPoint.cs]:
```suggestion
        /// The logger.
```

File: Emby.Server.Implementations/HttpServer/HttpResultFactory.cs [hunk 14]
```diff
@@ -24,12 +24,12 @@
...
 {
     /// <summary>
-    /// Class HttpResultFactory
+    /// Class HttpResultFactory.
     /// </summary>
     public class HttpResultFactory : IHttpResultFactory
     {
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/HttpServer/HttpResultFactory.cs]:
```suggestion
        /// The logger.
```

File: Emby.Server.Implementations/HttpServer/StreamWriter.cs [hunk 15]
```diff
@@ -10,37 +10,20 @@
...
-        /// Gets or sets the source stream.
-        /// </summary>
-        /// <value>The source stream.</value>
-        private Stream SourceStream { get; set; }
-
-        private byte[] SourceBytes { get; set; }
-
-        /// <summary>
-        /// The _options
+        /// The _options.
```
Comment (dkanada) [inline Emby.Server.Implementations/HttpServer/StreamWriter.cs]:
```suggestion
        /// The options.
```

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs [hunk 16]
```diff
@@ -7,68 +7,31 @@
...
-
-        /// <summary>
-        /// The logger
+        /// The logger.
         /// </summary>
         private readonly ILogger _logger;
 
         /// <summary>
-        /// The _json serializer
+        /// The _json serializer.
```
Comment (dkanada) [inline Emby.Server.Implementations/HttpServer/WebSocketConnection.cs]:
```suggestion
        /// The json serializer.
```

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs [hunk 17]
```diff
@@ -7,68 +7,31 @@
...
-
-        /// <summary>
-        /// Gets the last activity date.
-        /// </summary>
-        /// <value>The last activity date.</value>
-        public DateTime LastActivityDate { get; private set; }
-
-        /// <summary>
-        /// Gets the id.
+        /// The _socket.
```
Comment (dkanada) [inline Emby.Server.Implementations/HttpServer/WebSocketConnection.cs]:
```suggestion
        /// The socket.
```

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs [hunk 18]
```diff
@@ -105,10 +71,54 @@ public WebSocketConnection(IWebSocket socket, string remoteEndPoint, IJsonSerial
...
-            socket.Closed += socket_Closed;
+            socket.Closed += OnSocketClosed;
         }
 
-        void socket_Closed(object sender, EventArgs e)
+        /// <inheritdoc />
+        public event EventHandler<EventArgs> Closed;
+
+        /// <summary>
+        /// Gets or sets the _remote end point.
```
Comment (dkanada) [inline Emby.Server.Implementations/HttpServer/WebSocketConnection.cs]:
```suggestion
        /// Gets or sets the remote end point.
```

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 19]
```diff
@@ -36,19 +38,19 @@
...
 {
     /// <summary>
-    /// Class UserManager
+    /// Class UserManager.
     /// </summary>
     public class UserManager : IUserManager
     {
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/UserManager.cs]:
```suggestion
        /// The logger.
```

File: Emby.Server.Implementations/Library/Validators/ArtistsValidator.cs [hunk 20]
```diff
@@ -12,17 +12,17 @@
...
 {
     /// <summary>
-    /// Class ArtistsValidator
+    /// Class ArtistsValidator.
     /// </summary>
     public class ArtistsValidator
     {
         /// <summary>
-        /// The _library manager
+        /// The _library manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/ArtistsValidator.cs]:
```suggestion
        /// The library manager.
```

File: Emby.Server.Implementations/Library/Validators/ArtistsValidator.cs [hunk 21]
```diff
@@ -12,17 +12,17 @@
...
         /// </summary>
         private readonly ILibraryManager _libraryManager;
 
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/ArtistsValidator.cs]:
```suggestion
        /// The logger.
```

File: Emby.Server.Implementations/Library/Validators/GenresValidator.cs [hunk 22]
```diff
@@ -7,19 +7,28 @@
...
 {
-    class GenresValidator
+    /// <summary>
+    /// Class GenresValidator.
+    /// </summary>
+    public class GenresValidator
     {
         /// <summary>
-        /// The _library manager
+        /// The _library manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/GenresValidator.cs]:
```suggestion
        /// The library manager.
```

File: Emby.Server.Implementations/Library/Validators/GenresValidator.cs [hunk 23]
```diff
@@ -7,19 +7,28 @@
...
         /// </summary>
         private readonly ILibraryManager _libraryManager;
         private readonly IItemRepository _itemRepo;
 
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/GenresValidator.cs]:
```suggestion
        /// The logger.
```

File: Emby.Server.Implementations/Library/Validators/MusicGenresPostScanTask.cs [hunk 24]
```diff
@@ -8,12 +8,12 @@
...
 {
     /// <summary>
-    /// Class MusicGenresPostScanTask
+    /// Class MusicGenresPostScanTask.
     /// </summary>
     public class MusicGenresPostScanTask : ILibraryPostScanTask
     {
         /// <summary>
-        /// The _library manager
+        /// The _library manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/MusicGenresPostScanTask.cs]:
```suggestion
        /// The library manager.
```

File: Emby.Server.Implementations/Library/Validators/MusicGenresValidator.cs [hunk 25]
```diff
@@ -7,19 +7,28 @@
...
 {
-    class MusicGenresValidator
+    /// <summary>
+    /// Class MusicGenresValidator.
+    /// </summary>
+    public class MusicGenresValidator
     {
         /// <summary>
-        /// The _library manager
+        /// The _library manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/MusicGenresValidator.cs]:
```suggestion
        /// The library manager.
```

File: Emby.Server.Implementations/Library/Validators/MusicGenresValidator.cs [hunk 26]
```diff
@@ -7,19 +7,28 @@
...
         /// </summary>
         private readonly ILibraryManager _libraryManager;
 
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/MusicGenresValidator.cs]:
```suggestion
        /// The logger.
```

File: Emby.Server.Implementations/Library/Validators/StudiosValidator.cs [hunk 27]
```diff
@@ -9,19 +9,29 @@
...
 {
-    class StudiosValidator
+    /// <summary>
+    /// Class StudiosValidator.
+    /// </summary>
+    public class StudiosValidator
     {
         /// <summary>
-        /// The _library manager
+        /// The _library manager.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/StudiosValidator.cs]:
```suggestion
        /// The library manager.
```

File: Emby.Server.Implementations/Library/Validators/StudiosValidator.cs [hunk 28]
```diff
@@ -9,19 +9,29 @@
...
         /// </summary>
         private readonly ILibraryManager _libraryManager;
 
         private readonly IItemRepository _itemRepo;
+
         /// <summary>
-        /// The _logger
+        /// The _logger.
```
Comment (dkanada) [inline Emby.Server.Implementations/Library/Validators/StudiosValidator.cs]:
```suggestion
        /// The logger.
```

**Pull request merged**